ensdomains/ens-app-v3
# feat(wip): use existing commit if available

changes:
- added commit search

how to test:
1. go through registration process but do not start the commit tx
2. close the commit transaction modal
3. commit hash should be logged in the console
4. using a script like [this](https://gist.github.com/TateB/bc01f2a9b10a2b35c16dbcd9d997a84b), or via etherscan - run the commit transaction externally to the app
5. the app should automatically detect and update the status of the registration flow to be at the registration, and the timer should start
